kft-2020/README.md

```diff
@@ -0,0 +1,21 @@
+# Kubernetes Fest Tokyo 2020: Kubernetes Upstream Training
+
+<!--
+![Kubernetes Fest Tokyo 2020](./images/xxx.png)
+-->
+
+Welcome to Kubernetes Upstream Training!!
+
+This is the location of our Kubernetes Fest Tokyo 2020: Kubernetes Upstream Training (June 13th 2020) activities.
+
+## 資料 (Documents)
+
+* [参加者の皆様への宿題 Attendee prerequisites (in preparation)](docs/attendee-prerequisites.md)
+* [スライド Slides](docs/k8s-upstream-training-kft-2020.pdf)
```

Comment:
@atoato88 Are you planning to add this document later ?

Reply:
@ameukam Yes, I'll add this document later. Should I comment out here?

Reply:
Just wanted to be sure about what you're planning to do later. :-)